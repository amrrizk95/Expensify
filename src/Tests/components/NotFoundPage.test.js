import React from 'react'
import {NotFoundPage} from '../../components/NotFoundPage';
import { configure } from 'enzyme';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
test('should render Not found page',()=>{
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
})