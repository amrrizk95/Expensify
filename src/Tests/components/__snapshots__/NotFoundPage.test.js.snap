// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Not found page 1`] = `ShallowWrapper {}`;
